static/script/events/event.js

```diff
@@ -115,10 +116,11 @@ require.def('antie/events/event',
 			 * @param {Function} func The handler to be removed.
 			 */
 			removeEventListener: function(ev, func) {
```

Comment:
Hi Bas,

I've got to the bottom of the problem. Our app is calling removeEventListener where no event has been added.

We will be fixing this in our app, but could you also please add some error handling around this? We would expect a guard statement at the top of this function along the lines of:

```javascript
                if (!this._eventListeners[ev]) {
                    RuntimeContext.getDevice().getLogger().error("Attempting to remove non-existent event listener");
                    return;
                }
```

Thanks,

Tom